tools/coverage/githubUtil/github_test.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 func TestFilePathProfileToGithub(t *testing.T) {
-	input := "github.com/myRepoOwner/myRepoName/pkg/ab/cde"
+	input := "knative.dev/test-infra/pkg/ab/cde"
```

Comment:
It would be better to test both scenarios to ensure there is no regression.

Reply:
I don't think we can test both scenarios. GetRepoPath is making system calls it uses actual system env vars and directory.

Reply:
Not a problem. You can override GOPATH or mock GetRepoPath.